hypnos/src/event_handler.c

```diff
@@ -126,17 +127,21 @@ void button_pressed_isr(struct device *gpiobtn, struct gpio_callback *cb, uint32
 		bt_on();
 	}
 	gfx_update();
+	backlight_enable(true);
```

Comment:
Yeah, that's better

Reply:
On second thought, this appears to make the GUI feel less responsive. I prefer to have the screen light up fast, even if some data and graphics have not yet been updated. Please compare yourself by putting `backlight_enable(true);` before `clock_increment_local_time();`